tests/json/test_poetry_schema.py

```diff
@@ -4,7 +4,7 @@
 
 
 @pytest.fixture
-def base_object():
+def base_object() -> dict:
```

Comment:
this syntax is python 3.9+. less than that, the correct return type would be `Dict`, rather than `dict`.

I guess we can get away with it, since `mypy` is being run with a recent interpreter, and the type is ignored during other testing, but whichever way we go, it should be consistent throughout the codebase

Reply:
while we're at it, these `dict` annotations are equivalent to `Dict[Any, Any]`- we should be able to do better than that

Reply:
As you find out `dict` is valid as well, but don't let us define how keys and values look like in python<3.9. I agree it's in inconsistent use if we use `Dict` in other places, so I changed it.

> while we're at it, these dict annotations are equivalent to Dict[Any, Any]- we should be able to do better than that

The only way to do it better here is to use `Dict[str, Any]` (which doesn't add much benefit) or use `TypedDict`. But this is only available for python >=3.8 and would require to add `typing-extensions` as dependency for python <3.8. Not sure if we want to introduce another dependency.